### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by Jonny11bonk** [Tyrrrz/YoutubeDownloader#234] [General]
I have a "feature request". What if the downloading splitted into multiple parts for a faster downloading and then merged? This would be much faster than downloading into a single file. Also I would like to see making .m3u files or other playlist files to play the songs in that row.

**Comment by derech1e** [comment]
I dont think this feature is really useful. Your download speed is limited by your provider and splitting the downloads wont accelerate your downloads. The effort-benefit gab is to big in my oppinion.
